src/main/java/org/kohsuke/github/GHWorkflowRunJob.java

```diff
@@ -0,0 +1,256 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.commons.lang3.StringUtils;
+import org.kohsuke.github.GHWorkflowRun.Conclusion;
+import org.kohsuke.github.GHWorkflowRun.Status;
+import org.kohsuke.github.function.InputStreamFunction;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * A workflow run job.
+ *
+ * @author Guillaume Smet
+ */
+public class GHWorkflowRunJob extends GHObject {
```

Comment:
I thought `GHJob` would be a bit too generic but I can change it if you prefer it.

Reply:
The GitHub docs refer to ["Workflow Jobs"](https://docs.github.com/en/rest/reference/actions#workflow-jobs), but I'm also okay with "Workflow Run Jobs".

Reply:
Yeah, I wanted to make it clear they are associated with a run.

Reply:
I thought about it a bit more and `Workflow job` is less of a mouthful so I will switch to it.